spec/functions/hash_values_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'hash_values' do
+  # please note that these tests are examples only
+  # you will need to replace the params and return value
+  # with your expectations
+  it { is_expected.to run.with_params({}).and_return([]) }
+  it { is_expected.to run.with_params({ 'key' => 'value' }).and_return(['value']) }
+  it { is_expected.to run.with_params({ 'key' => { 'key1' => 'value1', 'key2' => 'value2' } }).and_return(['value1', 'value2']) }
```

Comment:
Maybe also a test to verify the ordering does not change and the same keys at different levels are not merged / overwritten, e.g.

```
  it { is_expected.to run.with_params({ 'key1' => 'value1', 'key2' => { 'key1' => 'value21', 'key2' => 'value22' }, 'key3' => 'value3' }).and_return(['value1', 'value21', 'value22', 'value3']) }
```

Reply:
added spec, Thanks for your review comment.